pkg/performanceprofile/cmd/render/render.go

```diff
@@ -58,14 +65,107 @@ func init() {
 	utilruntime.Must(performancev2.AddToScheme(manifestScheme))
 	utilruntime.Must(apicfgv1.Install(manifestScheme))
 	utilruntime.Must(mcfgv1.Install(manifestScheme))
+	utilruntime.Must(tunev1.AddToScheme(manifestScheme))
 	codecFactory = serializer.NewCodecFactory(manifestScheme)
 	runtimeDecoder = codecFactory.UniversalDecoder(
 		performancev2.GroupVersion,
 		apicfgv1.GroupVersion,
 		mcfgv1.GroupVersion,
+		tunev1.SchemeGroupVersion,
 	)
 }
 
+func expandCpuSet(cpuSet string) string {
```

Comment:
it's a draft, but please let's use the `cpuset` package (or one of its existing variants)

Reply:
Thanks for the comment, fixed